src/db/schema.rs

```diff
@@ -1,6 +1,7 @@
 table! {
+
     use diesel::sql_types::*;
-    use crate::sql_types::*;
+    //use crate::db::sql_types::*;
```

Comment:
We should change this in diesel.toml. Check the `import_types` option

Reply:
ok, should be good to go now!